src/store/sagas/send_and_receive.ts

```diff
@@ -61,9 +62,12 @@ export const fetchFeeRatesWatcher = createWatcher(fetchFeeRatesWorker, FETCH_FEE
 
 function* fetchExchangeRatesWorker() {
   try {
-    const { exchangeRates } = yield call(Relay.fetchFeeAndExchangeRates);
-    if (!exchangeRates) console.log('Failed to fetch exchange rates');
-    else yield put(setExchangeRates(exchangeRates));
+    const res = yield call(axios.get, EX_RATE_API);
+    if (res.status === 200 && res.data?.rates) {
+      const exchangeRates = reformatExchangeRates(res.data.rates);
+      yield put(setExchangeRates({ exchangeRates }));
+    } else console.log('Failed to fetch exchange rates');
```

Comment:
On failure we should let the user know with a toast or some notification that exchange rates were not loaded successfully and may be wrong/ outdated